tests/test_bot.py

```diff
@@ -244,6 +246,71 @@ async def test_no_token_passed(self):
         with pytest.raises(InvalidToken, match="You must pass the token"):
             Bot("")
 
+    def test_base_url_parsing_basic(self, offline_bot, caplog):
+        with caplog.at_level(logging.DEBUG):
+            bot = Bot(
```

Comment:
I totally forgot how the `offline_bot` fixture is used. I don't see it being referenced in these tests, would that still work?

Reply:
you're right, it's not used. I make the bot offline via the `OfflineRequest`. Removing.